src/js/components/devices/dialogs/make-gateway-dialog.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+
+// material ui
```

Comment:
remove?

Reply:
Nah - these are in many of the `js` files to signal the material ui related dependency group + are removed in the build step in any case 👌